source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/sector/Sector.java

```diff
@@ -91,6 +94,7 @@ public OPSSourceCharacteristics getDefaultCharacteristics() {
   }
 
   @Override
+  @JsonIgnore
```

Comment:
Wouldn't it be fine to just serialize it? In case we ever do get a different name/description combination...

Reply:
Yeah it would be fine, the parser-generator gizmo currently produces a warning when there is a getter without an associated field and I figured I'd just get rid of all the warnings. Will remove this one.